<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="1" hostname="PHX-ISHARAF" name="com.crm.qa.testcases.BranchOpportunityTest" tests="7" failures="0" timestamp="05 Dec 2019 17:16:47 GMT" time="451.685" errors="0">
  <testcase name="opttyClosedLost_viaAPI" time="45.709" classname="com.crm.qa.testcases.BranchOpportunityTest">
    <skipped/>
  </testcase> <!-- opttyClosedLost_viaAPI -->
  <testcase name="opttyClosedLost_viaAPI" time="57.541" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="logacall_notReached_Optty" time="62.550" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="logacall_Reached_StageClosedLost" time="71.329" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="moveStageStatusManually" time="54.793" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="scheduleMeetingandthenCloseMeeting" time="67.817" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
  <testcase name="scheduleMeetingviaNextAction" time="91.946" classname="com.crm.qa.testcases.BranchOpportunityTest"/>
</testsuite> <!-- com.crm.qa.testcases.BranchOpportunityTest -->
